docs/guide.mdx

```diff
@@ -61,10 +61,12 @@ If an access token isn't provided, the following rate limits apply:
 The Wave does not support the use of sha256 digest in the image name, e.g. `ubuntu@sha256:3235...ce8f`, when using
 the augmentation process to extend container images.
 
-In order to reference a container via sha256 digest in the image name with Wave you will need to *freeze* image mode
-that will force the creation of a new container image using the container you have specified as base image.
+While using a digest directly isn't possible, Wave can generate image names which include a sha256 digest
+of the `Dockerfile` used to build the image within the tag.
```

Comment:
I think this is confusing, the hash generated by Wave is not a sha256 checksum and it has nothing to do with the image digest

Reply:
```suggestion
```

Reply:
Removed 👍🏻